types.go

```diff
@@ -0,0 +1,36 @@
+package safecast
+
+// This files is highly inspired from https://pkg.go.dev/golang.org/x/exp/constraints
+// I didn't import it as it would have added an unnecessary dependency
+
+// Signed is an alias for all signed integers: int, int8, int16, int32, and int64 types.
+type Signed interface {
+	~int | ~int8 | ~int16 | ~int32 | ~int64
+}
+
+// Unsigned is an alias for all unsigned integers: uint, uint8, uint16, uint32, and uint64 types.
+// TODO: support uintpr
```

Comment:
it's really ez

https://github.com/ldemailly/go-scratch/commit/4608bec5f1ff7a31c85da80072858427006254b4

Reply:
I wasn't worried 😁 but thanks